tests/integration/test_keeper_three_nodes_start/test.py

```diff
@@ -3,26 +3,28 @@
 import helpers.keeper_utils as keeper_utils
 from helpers.cluster import ClickHouseCluster
 
-cluster = ClickHouseCluster(__file__)
-node1 = cluster.add_instance(
-    "node1", main_configs=["configs/enable_keeper1.xml"], stay_alive=True
-)
-node2 = cluster.add_instance(
-    "node2", main_configs=["configs/enable_keeper2.xml"], stay_alive=True
-)
 
 
-def get_fake_zk(nodename, timeout=30.0):
+
+def get_fake_zk(cluster, nodename, timeout=30.0):
     return keeper_utils.get_fake_zk(cluster, nodename, timeout=timeout)
 
 
 def test_smoke():
+    cluster = ClickHouseCluster(__file__)
```

Comment:
This will erase the whole instances dir, thus in case of repeated tests if one of the tests fail, we will lose all information about it. Maybe you pass a randomly generated string or a uuid as the `name` argument for `ClickHouseCluster` to separate the instances dir. 

```suggestion
    run_uuid = uuid.uuid4()
    cluster = ClickHouseCluster(__file__, str(run_uuid))
```

you need to `import uuid` at the beginning though.